test/integration_test/migration_features_test.go

```diff
@@ -330,9 +349,328 @@ func testMigrationStashStrategyWithStartApplication(t *testing.T) {
 	if err != nil {
 		logrus.Errorf("Error deleting namespace %s in source: %v\n", appData.Ns, err)
 	}
-
+	time.Sleep(30 * time.Second)
```

Comment:
Wondering if we do not need to make sure resources are cleaned up before exiting from this test?

Reply:
yes, will add that in another PR. The setups and cleanups part for these tests are mainly from operator tests. We have to add for all these tests.